### PR TITLE
Added Query #2 - List of IPs and Domains file communicated with

### PR DESCRIPTION
**Pull request by jing-microsoft**
Shows you a list of distinct IP addresses and DNS names the endpoint had network communication with through a specific file.

### CODE REVIEW

File: General queries/File footprint.txt [hunk 1]
```diff
@@ -10,3 +10,13 @@ project ComputerName, ActionType, FileName, InitiatingProcessFileName, EventTime
...
 | sort by ComputerName, LastEventTime desc
+
+// Query # 2 - Shows you a list of distinct IP addresses and DNS names the endpoint had network communication with through a specific file. 
+// Use this list to whitelist/blacklist IP addresses or understand if there are communication with IP you are not aware of.
+// Update the filename to the name you wish to investigate network communication.
+let filename = "FILENAME GOES HERE";
+// Builds table for distinct URLs based off filename
+NetworkCommunicationEvents 
+| where InitiatingProcessFileName =~ filename and ( isnotempty(RemoteIP) or isnotempty(RemoteUrl) )
+| project filename=InitiatingProcessFileName, DNS=RemoteUrl, IP=RemoteIP
```
Comment (TomerAlpert) [inline General queries/File footprint.txt]: I don't think you need this line. I'm sure you don't need to project filename.

Comment (jing-microsoft) [reply General queries/File footprint.txt]: Agreed. please remove

**Comment by jing-microsoft** [comment]
Do I close this pull request to complete this process  ?

**Comment by TomerAlpert** [comment]
No, I will.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jing-microsoft** [comment]
Oh yeah that's right.

**Pull request merged**